doc/source/whatsnew/v3.0.0.rst

```diff
@@ -375,7 +375,7 @@ Period
 
 Plotting
 ^^^^^^^^
--
+- Bug in  :meth:`DataFrameGroupBy.boxplot` passes a ``tuple`` instead of a ``string`` when input ``DataFrame`` is pre-grouped using more than one ``column`` (:issue:`14701`)
```

Comment:
The link to the API docs won't resolve unless you include a `.` before `DataFrameGroupBy`; so it should be `` :meth:`.DataFrameGroupBy.boxplot` ``